incubator/virtualcluster/pkg/syncer/resources/secret/dws.go

```diff
@@ -71,31 +69,30 @@ func (c *controller) Reconcile(request reconciler.Request) (reconciler.Result, e
 			return reconciler.Result{Requeue: true}, fmt.Errorf("There are pSecrets that represent vSerect %s/%s but the UID is unmatched.", request.Namespace, request.Name)
 		}
 	} else {
-		// We need to use name to search again for normal vScrect
+		// We need to use name to search again for normal vSecret
 		pSecret, err = c.secretLister.Secrets(targetNamespace).Get(request.Name)
```

Comment:
when sa type vSecret not exists, using secret name would get the one created by token controller. drop it.